variables.env

```diff
@@ -28,7 +29,11 @@ MEDIAWIKI_VERSION=1.41.1
 # as per https://github.com/wikimedia/wikidata-query-rdf/tags
 # and https://archiva.wikimedia.org/repository/releases/org/wikidata/query/rdf/service/
 # Compare to previous version: https://github.com/wikimedia/wikidata-query-rdf/compare/query-service-parent-0.3.135...query-service-parent-0.3.137
+# TODO: Add our own (semantic) version to hide this one?
 WDQS_VERSION=0.3.137
+# TODO: Decide on the independent versions to start from
+WDQS_FRONTED_VERSION=$WMDE_RELEASE_VERSION
```

Comment:
Here is my suggestion for this release: 

* 1.0.0 for all images that are attributed to the updates to the 1.39 release, including "deployment kit" for that series
* 2.0.0 for all images that are attributed to the updates to the 1.41 release, including "deployment kit" for that series
* 3.0.0 for all images that are attributed to the new 1.42 release, including "deployment kit" for that series

Reply:
Would you also have different major versions for all images, even though there might be no breaking change in between them? Such as WDQS based on 0.3.135 and 0.3.137?

Reply:
My suggestion here is only for this release, for the initial versions of everything. Updates from there, on all components of the system, strictly follow smear. So if I'm understanding your questions correctly, my answer would be no. When there is a patch update to WDQS and we make a new WBS WDQS image release, it gets a patch semvar bump (e.g. 1.0.0 to 1.0.1). Does that make sense?

Reply:
Perhaps what you're pointing out is that there may be no material difference between the WDQS image for 1.39 and the WDQS image for 1.42. If that is the case, then in what I'm suggesting is that the WBS WDQS image is only build once and remains v1.0.0 regardless, and the patch update would be 1.0.1.

Reply:
Yes, this is what I had in my head. So we would ship WDQS 1.0.0 with the 1.39 based Deployment Kit and maybe 1.0.1 with the 1.42 based Deployment Kit, given that there is just a "patch" change in WDQS between them? 

Or would you - for the first releases - still give individual major versions numbers to the WDQS versions of the different Deployment Kit versions?

Reply:
> then in what I'm suggesting is that the WBS WDQS image is only build once

I think our build system is not there yet.

Reply:
Yes, same paged. Thanks for clarification.

> but we should be fine as long as rebuilding the same version comes up with the same thing each time

Kind of, I still expect one weirdness here, see my comment https://github.com/wmde/wikibase-release-pipeline/pull/686#issuecomment-2160222671